crates/phactory/src/prpc_service.rs

```diff
@@ -478,47 +479,6 @@ impl<Platform: pal::Platform> Phactory<Platform> {
         Ok(fit_size(messages, output_buf_len))
     }
 
-    fn send_coordinate_info(&mut self, request: pb::SendCoordinateInfoRequest) -> RpcResult<()> {
-        // local ecdh_key
```

Comment:
Isn't this change belong to #498 ?

Reply:
I cannot compile if I don't remove it. Do you suggest to checkout an older version of the proto repo?

Reply:
Rolled back to an earlier version